test/e2e/tests/single_zone_pd_extreme_e2e_test.go

```diff
@@ -272,4 +272,64 @@ var _ = Describe("GCE PD CSI Driver pd-extreme", func() {
 			Expect(gce.IsGCEError(err, "notFound")).To(BeTrue(), "Expected disk to not be found")
 		}()
 	})
+
+	It("Should complete entire disk lifecycle with underspecified volume ID", func() {
+		testContext := getRandomTestContext()
+
+		p, z, _ := testContext.Instance.GetIdentity()
+		client := testContext.Client
+		instance := testContext.Instance
+
+		volName, _ := createAndValidateUniqueZonalDisk(client, p, z, standardDiskType)
```

Comment:
should this be the extreme pd disk type?

Reply:
Yes. Done.

Reply:
🙉 Sorry, I'm coming from hg/fig, where amend automatically adds modified files.